app/models/survey.rb

```diff
@@ -0,0 +1,5 @@
+class Survey < ActiveRecord::Base
```

Comment:
Remember to stay focused on the view (the behavior, the outside-in-ness) and making it do what you want. You can always refactor the underlying code once you get it working the way you want it to.